api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java

```diff
@@ -500,6 +500,351 @@ public void findCriteriaAutoCompleteExceptions() {
         "Bad Request: Please provide a valid type. blah is not valid.");
   }
 
+  @Test
+  public void findCriteriaByDomainPhysicalMeasurement() {
+    DbCriteria dbCriteria =
+        DbCriteria.builder()
+            .addCode("12345")
+            .addCount(10L)
+            .addConceptId("123")
+            .addDomainId(Domain.PHYSICAL_MEASUREMENT_CSS.toString())
+            .addGroup(Boolean.FALSE)
+            .addSelectable(Boolean.TRUE)
+            .addName("chol blah")
+            .addParentId(0)
+            .addType(CriteriaType.PPI.toString())
+            .addAttribute(Boolean.FALSE)
+            .addStandard(false)
+            .addFullText("[PHYSICAL_MEASUREMENT_CSS_rank1]")
+            .build();
+    cbCriteriaDao.save(dbCriteria);
+
+    assertThat(cohortBuilderMapper.dbModelToClient(dbCriteria))
+        .isEqualTo(
+            controller
+                .findCriteriaByDomain(
+                    WORKSPACE_NAMESPACE,
+                    WORKSPACE_ID,
+                    Domain.PHYSICAL_MEASUREMENT_CSS.toString(),
+                    false,
+                    "12345",
+                    null,
+                    null)
+                .getBody()
+                .getItems()
+                .get(0));
+
+    assertThat(
+            controller
+                .findCriteriaByDomain(
+                    WORKSPACE_NAMESPACE,
+                    WORKSPACE_ID,
+                    Domain.PHYSICAL_MEASUREMENT_CSS.toString(),
+                    true,
+                    "12345",
+                    null,
+                    null)
+                .getBody()
+                .getItems())
+        .isEmpty();
```

Comment:
Instead of checking that the standard flag returns empty in each test, it would be more efficient to write 1 test that proves this. It would also reduce lines of code.

Reply:
All the scenarios? Lets discuss this offline